src/index.test.js

```diff
@@ -189,4 +201,35 @@ describe('LazyRender', () => {
       });
     });
   });
+
+
+  describe('Get Container', () => {
+    it('should return window when scrollparent returns body in CSS1Compat mode', () => {
+      mockCompatMode('CSS1Compat');
+      mockScrollParent = document.body;
+
+      const element = wrapper(
+        <LazyRender placeholder={placeholder} content={content}>
+          {() => children}
+        </LazyRender>
+      );
+      expect(element.instance().getContainer()).toBe(window);
```

Comment:
rather than testing via instance/internal methods, could we check the value that the render fn gets called with when in the relevant compat mode?

e.g.https://github.com/jameslnewell/react-lazily-render/blob/master/src/index.test.js#L97

Reply:
I don't know if there is a good way to do this for this case? We need to check which element is being returned to bind the events to, and that info doesn't get sent through to the render function

Reply:
@jameslnewell How can we do so? The callback just give a boolean right?

Reply:
Yeah it just gives you a boolean, but the addition of the compat mode handling should result in the boolean being true under additional conditions right?

Reply:
True, we'd have to change how we mock the `getViewportBounds()` method. I don't think its insurmountable but for the moment I've changed the tests to assert on the add/remove listener methods.